test/box_test.dart

```diff
@@ -1,64 +1,72 @@
-import "package:test/test.dart";
-import "package:objectbox/objectbox.dart";
-import "entity.dart";
+import 'package:test/test.dart';
+import 'package:objectbox/objectbox.dart';
+import 'entity.dart';
 import 'test_env.dart';
 
+// ignore_for_file: omit_local_variable_types
```

Comment:
kinda, it's also because having the types is actually part of the test - so that the return types of some functions are the right types, maybe I'll add this info as a comment

Reply:
added the comment